inst-support-eng/panda-pto
# Refactor csv imports

test plan
- run migrations
- navigate to admin page, import agents.csv
(note: agents.csv file should create new user(s) and update existing user)
- verify affected user objects update/create with all needed user values
- verify existing user data is still in tact 

closes #72
